src/types/quizzes.d.ts

```diff
@@ -1,11 +1,31 @@
-import { ConstructorClientOptions, NetworkParameters, UserParameters } from '.';
+import { Nullable } from './index.d';
+import {
+  ConstructorClientOptions,
+  Facet,
+  Feature,
+  Group,
+  NetworkParameters,
+  RequestFeature,
+  RequestFeatureVariant,
+  SortOption,
+  FilterExpression,
+  ResultSources,
+} from '.';
 
 export default Quizzes;
 
 export interface QuizzesParameters {
   section?: string;
   answers?: any[];
```

Comment:
Not related to this PR but can we make this required. It's already required in the implementation